packages/matchbox/src/components/Banner/Banner.js

```diff
@@ -12,35 +12,40 @@ import { buttonReset } from '../../styles/helpers';
 import { buttonFrom } from '../Button';
 import { margin } from 'styled-system';
 
-function IconSection({ status }) {
+function IconSection({ status, size }) {
   const statusIcon = React.useMemo(() => {
     return status === 'default' ? statusIcons.info : statusIcons[status];
   }, [statusIcons, status]);
 
   const Icon = statusIcon.iconName;
 
+  const iconSize = size === 'large' ? '3rem' : '1rem';
+  const bgColor = size === 'large' ? statusIcon.bg : null;
+  const fillColor = size === 'large' ? statusIcon.fill : statusIcon.bg;
```

Comment:
The yellow banner's icon is a bit light, not sure if theres an easy way to add a darker yellow for one